src/xcode/ENA/ENA/Source/Scenes/Events/Cells/AddEventTableViewCell.swift

```diff
@@ -35,7 +35,7 @@ class AddEventTableViewCell: UITableViewCell {
 
 	// MARK: - Internal
 
-	func configure(cellModel: AddEventCellModel) {
+	func configure(cellModel: AddCellModeling) {
```

Comment:
Would it make sense to rename this cell to `AddTableViewCell` then?

Reply:
Yes, we can :)